src/renderer/actions/install.js

```diff
@@ -100,18 +114,28 @@ async function injectShims(paths) {
     const progressPerLoop = (INJECT_SHIM_PROGRESS - progress.value) / paths.length;
     for (const discordPath of paths) {
         log("Injecting into: " + discordPath);
+        const appAsar = path.join(discordPath, "app.asar");
+        const discordAsar = path.join(discordPath, "discord.asar");
```

Comment:
These variables seem to be leftovers from the old asar-renaming injection method. Probably best to comment them.